### PR TITLE
parallelize next plugin test

### PR DESCRIPTION
**Pull request by bengl**
### What does this PR do?
See title
### Motivation
It's very slow otherwise (>10min)

### CODE REVIEW

**Comment by rochdev** [comment]
Any way to revert this PR instead? https://github.com/DataDog/dd-trace-js/pull/3935

There was already logic in place before to split not only by Node version but also by Next version which brought this down to ~2min.

**Comment by bengl** [comment]
> Any way to revert this PR instead? #3935
> 
> There was already logic in place before to split not only by Node version but also by Next version which brought this down to ~2min.

I'd rather not, because that leaves us with [two places](https://en.wikipedia.org/wiki/Magic_number_(programming)) where version ranges for this plugin are defined.

**Review by tlhunter — CHANGES_REQUESTED** [review]
> All checks have passed
> 36 successful checks

Shouldn't this number be over 100?

**Review by Qard — DISMISSED** [review]

File: .github/workflows/appsec.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ on:
   schedule:
     - cron: '0 4 * * *'
 
+env:
+  OLDEST_NODE: 18
```
Comment (simon-id) [inline .github/workflows/appsec.yml:11]: this shouldn't be merged to v4 no ?

Comment (simon-id) [reply .github/workflows/appsec.yml:11]: I mean dont we support node 16 in v4 ?

Comment (bengl) [reply .github/workflows/appsec.yml:11]: This just replaces some existing hard-coded instances of `18` in this file. Presumably we'd need to backport this to the v4 release branch separately, with `16` here, but I'm not sure what's currently on that branch. I haven't looked.

**Comment by bengl** [comment]
@tlhunter 

> Shouldn't this number be over 100?

Yep. Looks like the workflow file might be invalid. I'll fix that.

**Review by tlhunter — APPROVED** [review]

**Pull request merged**